about.html

```diff
@@ -0,0 +1,125 @@
+          <a href="./index.html"
+            ><img src="./images/FIAC-logo-2X.png" alt="fiac logo"
+          /></a>
+        </li>
+        <li class="home"><a href="./index.html">Home</a></li>
+        <li><a href="./index.html#organizers">Organizers</a></li>
+        <li><a href="./index.html#program">Program</a></li>
+        <li><a href="./index.html#speakers">Speakers</a></li>
+        <li><a href="./about.html">About</a></li>
+      </ul>
+    </nav>
+    <div class="bread-crumb"><p></p></div>
+    <header class="about">
+      <h2>"Ready for First Contact!"</h2>
+      <h1>FIRST INTERPLANETARY ARRIVAL CONFERENCE</h1>
+      <hr class="header-hr">
+      <div class="about-text">
+        <p>
+          Our first contact with outer world civilizations! Thanks to our partner agencies around the globe, we are able to make the first-ever extraterrestrial conference. They say they are coming in peace. They assure us they want to help us become a better civilization. Will they teach us new technologies? Will we be able to communicate with them through our expert interpreters? Are you ready? Sign up and hold on tight!
+        </p>
+      </div>
+      <div class="contact">
+        <p>
+          Please contact us per Email for any further questions about our First Interplanetary Arrival Conference 2022!
+        </p>
+        <p class="email">fiacs2022@globalsecurity.org</p>
+      </div>
+    </header>
+    <section class="about-logo">
+      <div>
+        <h2 class="the-logo">First Interplanetary Arrival Conference Logo</h2>
+      </div>
+      <hr class="logo-hr" />
+      <div class="text">
+        The logo of the First Interplanetary Arrival Conference was imposed by us by the Legal Ambassador of Atlkaya-Mirbukia Tuloska Galaxy Ring.
+      </div>
+      <div class="logo">
+        <img src="./images/FIAC-logo-2X.png" alt="FIAC logo" />
+      </div>
+    </section>
+    <hr class="between-sections" />
+    <section class="about-planning">
+      <div>
+        <h2 class="possible">How this event was made possible?</h2>
+      </div>
+      <hr class="planning-hr" />
+      <div class="text">
+        Take a look at the planning that went into the making of the one in a
+        lifetime event. First contact with extraterrestrial beings is no easy
+        task to plan!
+      </div>
+      <div class="videos-wrapper">
+        <div class="video v1">
+          <h2>First Contact</h2>
+          <p>When SETI Contacted NASA and Everything Began</p>
+        </div>
+        <div class="video v2">
+          <h2>UNSC Communication</h2>
+          <p>United Nations Security Council Planning with Intergallactic Beings</p>
+        </div>
+      </div>
+    </section>
+    <section class="organizers">
+      <h2>Organizers</h2>
+      <hr class="organizers-hr" />
+      <div class="organizers-wrapper">
+        <img
+          src="./images/sponsor-logos/china_national_space_administration_logo.png"
+          alt=""
+        />
```

Comment:
- [ ] [OPTIONAL] Please try to include an alternate text in every `img` tag to help with accessiblity.